### PR TITLE
Add onChange handler to ssm + audit packages

### PR DESCRIPTION
**Pull request by sdomagala**
Hello,
In this PR I'd like to add functionality to automatically trigger handler when ssm parameter is regenerated - we use this data in other parts of code and need to be able to regenerate if any parameter changed.

While at it, I saw multiple vulnerabilities doing npm audit so I thought it would be sensible to make it a part of pipeline so nothing slips to production anymore (added only to Node8.10 as 6.10 doesn't have that command yet) .

### Edit
Actually, npm audit is not available in Circle CI, had to revert this change

Cheers

### CODE REVIEW

File: src/middlewares/__tests__/ssm.js [hunk 1]
```diff
@@ -189,6 +217,39 @@ describe('🔒 SSM Middleware', () => {
...
+        },
+        cache: true,
+        onChange: onChange,
+        cacheExpiryInMillis: 10,
+        setToContext: true,
+        paramsLoaded: false
+      },
+      callbacks: [
+        (_, {context}) => {
+          expect(onChange.mock.calls.length).toEqual(1)
```
Comment (vladgolubev) [inline src/middlewares/__tests__/ssm.js]: Just FYI there is [.toHaveBeenCalledTimes(number)](https://jestjs.io/docs/en/expect#tohavebeencalledtimesnumber) API:)

Comment (sdomagala) [reply src/middlewares/__tests__/ssm.js]: I knew there must be something like this but couldn't find it - changed :)

**Comment by lmammino** [comment]
I love this feature :)
Thanks @sdomagala!

Once merged would you be willing to port this change to the 1.x branch?

**Comment by sdomagala** [comment]
Thanks for quick response 😉 
Holiday may be a bit busy but if I can squeeze an hour or two then I'll definitely do it!

**Pull request merged**